### DISCUSSION TITLE
[PSD1] Exercise 1.46

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#121] [Esercizi (A-L)]
**Exercise 1.46**

USB 3.0 can send data at 5 Gbits/sec. How many bytes can it send in 1 minute?

**Comment by Elia-Belli** [comment]
- 5 Gbits/s = 0.625 Gbytes/s = 37.5 Gbytes/min = 37_500_000_000 bytes/min
